ciao-4.11/contrib/lib/python3.5/site-packages/lightcurves.py

```diff
@@ -566,7 +566,17 @@ def plot(self, rateaxis="y", gcol="lime", erase=True):
         # setup figure
 
         if rateaxis == "y":
-            fig, axs = plt.subplots(2, 1, tight_layout=True)
+            from matplotlib import tight_layout
```

Comment:
@nplee - this still works with matplotlib 2 ? I'm guessing so but just want to check.

Reply:
yes it seems to work with matplotlib 2 from my limited testing.